addon/commands/_private/rdfa-commands/remove-properties-targeting-backlinks.ts

```diff
@@ -0,0 +1,61 @@
+import { Backlink } from '@lblod/ember-rdfa-editor/core/rdfa-processor';
+import { Command, Transaction } from 'prosemirror-state';
+import { getNodesByResource } from '@lblod/ember-rdfa-editor/plugins/rdfa-info';
+import {
+  getProperties,
+  getResource,
+} from '@lblod/ember-rdfa-editor/utils/_private/rdfa-utils';
+
+export const removePropertiesTargetingBacklinks = ({
```

Comment:
I would just rename this to either `removePropertyTargetingBacklink` or `removePropertiesTargetingBacklink`. Just to be consistent :)